<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>421</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_orienMainGraph"/>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stop">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_upVec">
   <property name="geometry">
    <rect>
     <x>462</x>
     <y>62</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>up vector</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_height">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>height set (0 - 200)</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_height">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>200</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>20</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_upVec_value">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>90</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>[0,0,0]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_height_value">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>220</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>current target:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_heading">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>250</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>heading set (0 - 360)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stopVessel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>400</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>stop vessel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_moveControl">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>330</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>move control</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_forward">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>360</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>forward</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>360</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_heading">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>280</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_heading_value">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>300</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>current heading:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_currentMode">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>130</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>current mode:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
